test/teiserver_web/tachyon/matchmaking_test.exs

```diff
@@ -427,7 +427,10 @@ defmodule Teiserver.Tachyon.MatchmakingTest do
 
       host_data = %{
         ips: ["127.0.0.1"],
-        port: 48912
+        port: 48912,
+        engine: "recoil",
+        game: "BAR",
+        map: "Altored Divide"
```

Comment:
This is incorrect. The response from the autohost does NOT have these keys set, only the ips and port. See https://github.com/beyond-all-reason/tachyon/blob/master/docs/schema/autohost.md#response-8

What you need to check is on lines 454, where we call `recv_message!` for each client, that *these* responses have the correct data in them.
You can also remove the comments about running schema validation of these responses, this is now done

Reply:
Done